cmd/sonobuoy/app/args_test.go

```diff
@@ -1,89 +0,0 @@
-/*
-Copyright the Sonobuoy contributors 2019
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package app
-
-import (
-	"testing"
-
-	"github.com/spf13/pflag"
-
-	ops "github.com/vmware-tanzu/sonobuoy/pkg/client"
-)
-
-func TestGetE2EConfig(t *testing.T) {
```

Comment:
Mode is just a flag setting env vars; we have gen tests and smoke tests etc. No loss of coverage imo.